Backend/UnitTesting/app.config

```diff
@@ -1,40 +1,25 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
-<configuration>
-  <configSections>

```

Comment:
should not have changed the app.config dependencies

Reply:
This app.config looks like the ones in the controller project lol

Reply:
We just have to make sure the public key for the DAL is in there